system-tests/README.md

```diff
@@ -178,7 +178,17 @@ Once completed, following services will start within their docker containers:
 
 (EDC Connectors will also be seeded with initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json))
 
-_Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts and `cli-tools` container will also automatically stop after registering participants._
+> Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts.
+
+> The container `cli-tools` will turn into the state `healthy` after registering successfully all participants.
+
+Sample for confirming successful run of container `cli-tools`.
+
+```powershell
+PS C:\> docker ps
```

Comment:
```suggestion
docker ps
```

Reply:
consider removing prompts from commands throughout the document, to make this clearer and more cross-shell.

Reply:
Agreed! Thanks for bringing this up!